### PR TITLE
Delay optimizer imports

### LINKED ISSUE #139: Delay optimizer imports
I had issues loading libraries that I wasn't even using, because the optimizers are being imported, regardless of use.

I suggest delaying those imports to the "if" scenarios that require them.

### PR DESCRIPTION
**Pull request by jedmao**
Fixes #139

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -68,15 +63,15 @@ module.exports = function(content) {
...
-      plugins.push(imageminSvgo(options.svgo));
+      plugins.push(require('imagemin-svgo')(options.svgo));
     if(options.pngquant.enabled !== false)
-      plugins.push(imageminPngquant(options.pngquant));
+      plugins.push(require('imagemin-pngquant')(options.pngquant));
     if(options.optipng.enabled !== false)
-      plugins.push(imageminOptipng(options.optipng));
+      plugins.push(require('imagemin-optipng')(options.optipng));
     // optional optimizers
     if(options.webp)
```
Comment (tcoopman) [inline index.js]: why didn't you add imagemin-webp?

Comment (jedmao) [reply index.js]: Old branch actually. I can add it.

Comment (jedmao) [reply index.js]: @tcoopman, fixed in 7cf2f22 (squashed)

Comment (tcoopman) [reply index.js]: The fix is not on this PR?

File: index.js [hunk 2]
```diff
@@ -68,18 +62,18 @@ module.exports = function(content) {
...
-      plugins.push(imageminWebp(options.webp));
+      plugins.push(require('imagemin-webp')(options.webp));
```
Comment (jedmao) [inline index.js]: Sure it is. It's right here.

File: index.js [hunk 3]
```diff
@@ -1,10 +1,4 @@
 var imagemin = require('imagemin');
-var imageminGifsicle = require('imagemin-gifsicle');
-var imageminMozjpeg = require('imagemin-mozjpeg');
-var imageminOptipng = require('imagemin-optipng');
-var imageminSvgo = require('imagemin-svgo');
-var imageminPngquant = require('imagemin-pngquant');
-var imageminWebp = require('imagemin-webp');
```
Comment (jedmao) [inline index.js]: And here.

Comment (tcoopman) [reply index.js]: Strange, didn't see them before, maybe some caching of the browser.
Thanks

**Comment by jedmao** [comment]
@tcoopman thanks for the merge! Can you cut a release for this, pretty please?

**Comment by jedmao** [comment]
Published as version 4.1.0. Thanks!

**Pull request merged**